index.php

```diff
@@ -1,37 +1,111 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <link href="conference.css" type="text/css" rel="stylesheet" />
-  </head>
-  <body>
-    <nav>
-      <ul class="navbar">
-        <li><a class="active" href="index.php">Events</a></li>
-        <li><a href="attendees.php">Attendees</a></li>
-        <li><a href="sponsors.php">Sponsors</a></li>
-        <li><a href="jobs.php">Jobs</a></li>
-        <li><a href="hotel.php">Hotel</a></li>
-        <li><a href="committees.php">Committees</a></li>
-        <li><a href="logistics.php">Logistics</a></li>
-      </ul>
-    </nav>
-    <header>
-      <!-- placeholder -->
-      <h1>stuff for events</h1>
+    <head>
+        <link href="./conference.css" type="text/css" rel="stylesheet" />
+        <script>
+        if (window.history.replaceState) {
+            window.history.replaceState(null, null, window.location.href);
+        }
+        
+        </script>
+    </head>
+    <body>
+        <nav>
+            <ul class="navbar">
+                <li><a class="active" href="index.php">Events</a></li>
+                <li><a href="attendees.php">Attendees</a></li>
+                <li><a href="sponsors.php">Sponsors</a></li>
+                <li><a href="jobs.php">Jobs</a></li>
+                <li><a href="hotel.php">Hotel</a></li>
+                <li><a href="committees.php">Committees</a></li>
+                <li><a href="logistics.php">Logistics</a></li>
+            </ul>
+        </nav>
+        <header>
+            <h1>Session Events</h1>
+        </header>
+        <div class="main">
+            <?php include 'pdo.php'?>
 
-    </header>
-    <div class="main">
-      <h2>list all events for a particular day</h2>
-      <h2>switch a session's day/time and/or location</h2>
-      <!-- placeholder -->
-      <?php
+            <div id="listDayEvents">
+                <h2>Event Date:</h2>
+                <select id='dateselect' onChange='displayEvent(this.value)'>
+                    <option value='saturday' selected='selected'>saturday</option>
```

Comment:
I think we need the empty option back; the selection is on 'Saturday' and there isn't a table for it when I refresh

Reply:
I don't know if AJAX would allow us to cache the query somehow, but i don't think it's worth the effort compared to just introducing an empty option